lib/pkgcloud/core/base/client.js

```diff
@@ -28,6 +28,30 @@ var Client = exports.Client = function (options) {
 util.inherits(Client, events.EventEmitter2);
 
 /**
+ * Client.setCustomUserAgent
+ *
+ * @description allows the caller to specify a custom prefix for the HTTP UserAgent
+ * for all queries generated during the lifetime of the client.
```

Comment:
Add a bit of documentation here to show the user the proper format for a user agent. It's really easy to get wrong.